composer.json

```diff
@@ -10,11 +10,12 @@
         "php": "^8.1",
         "google/protobuf": "^3.22",
         "php-http/discovery": "^1.14",
+        "psr/http-client": "^1.0",
         "psr/http-client-implementation": "^1.0",
         "psr/http-factory-implementation": "^1.0",
-        "psr/http-client": "^1.0",
         "psr/http-message": "^1.0.1|^2.0",
         "psr/log": "^1.1|^2.0|^3.0",
+        "ramsey/uuid": "^4.7",
```

Comment:
I think it'd be worth checking what is the oldest version this will work with, and making that the minimum.

Reply:
4.3.0 was released March 26, 2022 and looks to work correctly